Domain_Model.md

```diff
@@ -0,0 +1,60 @@
+## Domain Model
```

Comment:
Very nice use of the Domain model and formatting clearly with the actions vs properties